File: algorithmProblemSolving/programmers/level1/시저 암호.js
function solution(s, n) {
    var answer = '';
    return s.split('').map((v)=>{
        let code = v.charCodeAt(0);
        if(code===32) return v;
        if(code<91){
            return String.fromCharCode((code-65+n)%26+65);
        }else{
            return String.fromCharCode((code-97+n)%26+97);
        }
        return v;
    }).join('');
}

File: algorithmProblemSolving/programmers/level1/시저 암호.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;
vector<char> upperCharV;
vector<char> lowerCharV;

string solution(string s, int n) {
    string answer = "";
    for(char i='a';i<='z';i++){
        lowerCharV.push_back(i);
    }
    for(char i='A';i<='Z';i++){
        upperCharV.push_back(i);
    }
    for(int i=0;i<s.length();i++){
        if(s.at(i)==' '){
            answer = answer+' ';
        }else if(s.at(i)<91){
            answer = answer+upperCharV[(s.at(i)-'A'+n)%26];
        }else{
            answer = answer+lowerCharV[(s.at(i)-'a'+n)%26];
        }
    }
    return answer;
}